src/utils/format.js

```diff
@@ -84,6 +84,36 @@ export function formatDecodeByteArray(bytesArray) {
   return String.fromCharCode(...bytesArray)
 }
 
+export function formatTokenLimit(extensions, tokenLimit) {
+  if (extensions.includes('mintable') && extensions.includes('mintable_limit')) {
+    return tokenLimit
+  } else if (extensions.includes('mintable') && !extensions.includes('mintable_limit')) {
+    return 'Unlimited'
+  } else if (extensions.includes('mintable_templates') && extensions.includes('mintable_templates_limit')) {
+    return null
+  } else if (extensions.includes('mintable_templates') && !extensions.includes('mintable_templates_limit')) {
+    return null
+  }
+}
+
+export function formatTemplateLimit(extensions, templateLimit) {
+  if (extensions.includes('mintable') && extensions.includes('mintable_limit')) {
+    return null
+  } else if (extensions.includes('mintable') || !extensions.includes('mintable_limit')) {
+    return null
+  } else if (extensions.includes('mintable_templates') && extensions.includes('mintable_templates_limit')) {
+    return 'Unlimited'
```

Comment:
```suggestion
    return templateLimit
```

Should return templateLimit, according to the table https://github.com/aeternity/aescan/issues/145

Reply:
fixed